src/DemoApp.vue

```diff
@@ -12,6 +12,9 @@
     <form-radio-button-group name="sampleRadioButtonGroup" label="Sample Radio Button Group" :options="radioButtonOptions" helper="This shows rendering a radio button group with a required validation rule" validation="required"></form-radio-button-group>
     <form-checkbox name="sampleCheckbox" label="Sample Checkbox" v-model="data.sampleCheckbox" helper="This checkbox represents a boolean in the data model."></form-checkbox>
     Current Value: {{data.sampleCheckbox}}
+    <br/>
+    <br/>
+    <form-html-editor name="sampleHtmlText" label="Sample Html Editor" v-model="data.sampleHtmlText" validation="required|max:300" />
```

Comment:
Instead of adding line breaks, using margin would keep styling out of the markup and allow for greater flexibility. Take a look at https://bootstrap-vue.js.org/docs/reference/spacing-classes/. You could add `class="mt-3"` here.